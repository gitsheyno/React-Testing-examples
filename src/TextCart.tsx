export default function TextCart() {
  return <div>TextCart</div>;
}
