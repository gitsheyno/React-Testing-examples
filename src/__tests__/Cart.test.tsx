import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import TextCart from "../TextCart";

test("snapshot test for TextCart", () => {
  const { asFragment } = render(<TextCart />);
  expect(asFragment()).toMatchSnapshot();
});
