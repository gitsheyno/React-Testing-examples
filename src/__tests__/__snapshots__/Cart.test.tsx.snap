// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot test for TextCart 1`] = `
<DocumentFragment>
  <div>
    TextCart
  </div>
</DocumentFragment>
`;
